plaso/parsers/esedb_plugins/msie_webcache_cookies.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""Parser for the Microsoft Internet Explorer WebCache ESE database.
+
+The WebCache database (WebCacheV01.dat or WebCacheV24.dat) are used by MSIE
```

Comment:
Is this the same database as https://github.com/log2timeline/plaso/blob/main/plaso/parsers/esedb_plugins/msie_webcache.py ? then I opt to extend that parser instead of a separate one

Reply:
weird stuff happens when I tried to build them as a single plugin. `parser_mediator.ProductEventData` is reached, but the data never appears in the output. It shows up in the test output, as there are N new events (N:= number of cookies), but it doesn't appear in the psteal results.

If they are two separate plugins it works fine.

Reply:
disregard, it was an issue with timeliner.yaml